test/sharness/Makefile

```diff
@@ -14,6 +14,9 @@ IPFS_ROOT = ../..
 # User might want to override those on the command line
 GOFLAGS =
 
+# The targets in this makefile depend on being run sequentially
+.NOTPARALLEL:
```

Comment:
To run tests in parallel I think we need something like:

```
all:
	make clean
	make deps
	make $(T)
	make aggregate
```

and then use `make -j 4` for example.